File: tests/cases/global.py
#!/usr/bin/env python3

CODE_0 = 0
CODE_1 = 1

L_A = [CODE_0, CODE_1]

CODE_A = "a"
CODE_B = "b"

L_B = [CODE_A, CODE_B]

if __name__ == "__main__":
    for i in L_A:
        print(i)
    for j in L_B:
        print(j)
    # test for container membership
    if "a" in ["a", "b"]:
        print("OK")

File: tests/cases-py2many/global.jl
code_0 = 0
code_1 = 1
l_a = OffsetArray([code_0, code_1], -1)
code_a = "a"
code_b = "b"
l_b = OffsetArray([code_a, code_b], -1)
function main()
    for i in l_a
        println(i)
    end
    for j in l_b
        println(j)
    end
    if "a" in ["a", "b"]
        println("OK")
    end
end

main()
